pkg/diff/diff.go

```diff
@@ -969,10 +969,10 @@ func CreateTwoWayMergePatch(orig, new, dataStruct interface{}) ([]byte, bool, er
 	return patch, string(patch) != "{}", nil
 }
 
-// HideSecretData replaces secret data values in specified target, live secrets and in last applied configuration of live secret with stars. Also preserves differences between
+// HideSecretData replaces secret data & optional annotations values in specified target, live secrets and in last applied configuration of live secret with stars. Also preserves differences between
```

Comment:
I kinda object to `Hide` as a verb, this appears to be closer to `Mask` than `Hide`.

That appears to be a preexisting condition, but if I inherited this code, I'd want to change it.